src/herder/test/FeeBumpTests.cpp

```diff
@@ -0,0 +1,606 @@
+// Copyright 2019 Stellar Development Foundation and contributors. Licensed
+// under the Apache License, Version 2.0. See the COPYING file at the root
+// of this distribution or at http://www.apache.org/licenses/LICENSE-2.0
+
+#include "herder/TransactionQueue.h"
```

Comment:
do not call this file `FeeBumpTests.cpp` : it's the same filename used in the transactions test suite